app/build.gradle

```diff
@@ -23,8 +23,6 @@ android {
         logger.info("This could be a CI build on pre-merged branch so there is no ticket id found!")
     }
 
-    project.ext.set("versionNameSuffix", "$gitCommitHash${ticketValue}")
```

Comment:
Oh! One last thing 🙏 

Seems like the part above is not being used anywhere but in this line, I believe we can remove it?

```
 // Our common part of fetching the git hash commit then appending to versionName for internal build.
    def gitCommitHash = "git rev-parse --short HEAD".execute().text.trim()
    def gitCommitMessage = "git show -s --format=%B HEAD".execute().text.trim()
    def ticketValue = ""
    try {
        ticketValue = "." + (gitCommitMessage =~ "\\[(.*)\\]")[0][1]
    } catch (Exception e) {
        logger.info("This could be a CI build on pre-merged branch so there is no ticket id found!")
    }
```